### PR TITLE
Return React component

### PR DESCRIPTION
**Pull request by nicl**
Just to show how to serve a React component.

### CODE REVIEW

File: src/server.tsx [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 const PORT = 3030;
 
 const app = express();
 
 app.use(express.json({ limit: '50mb' }));
 
-app.get('/healthcheck', (req, res) => res.send({ msg: 'hello Andre, Tom and Nic!' }));
+app.get('/', (req, res) => {
+    const ExampleComponent: React.FC<{}> = () => <div>Foo</div>;
```
Comment (tjmw) [inline src/server.tsx]: Really nit-picky, but it feels like maybe this line where we define the component should be pulled out of this handler. I think we only really need to define it once, and then render when the request comes in. What do you reckon?

**Comment by nicl** [comment]
@andre1050 I was thinking we were going to include a library - we just weren't sure which one yet. I don't have a strong opinion on this though - I had a brief look at options yesterday but a lot of the runtime-free options require webpack/babel. We could just stick with Emotion which has served us well?

**Comment by andre1050** [comment]
> @andre1050 I was thinking we were going to include a library - we just weren't sure which one yet. I don't have a strong opinion on this though - I had a brief look at options yesterday but a lot of the runtime-free options require webpack/babel. We could just stick with Emotion which has served us well?

Oh cool, happy to go with Emotion. In we feel like we wanna compare some options, we can also have a look here - https://github.com/MicheleBertoli/css-in-js

**Comment by tjmw** [comment]
My $0.02 would be to go with Emotion since we're already using elsewhere and are happy with it. Unless anyone feels strongly that we should go a different route.

**Pull request merged**